test/test_files/function/gds/scc.test

```diff
@@ -0,0 +1,132 @@
+-DATASET CSV empty
+
+--
+
+-CASE Scc1
```

Comment:
What's Scc1 and Scc2 testing exactly?

Can you make sure your test graph covers the following SCCs: 

- completely isolated node n
- node that only has outgoing edges
- node that has only incoming edges
- a small completely isolated scc
- a small scc that only has outgoing edges
- a small scc that only has incoming edges

Reply:
Refactored the tests to add all these cases.